python_modules/dagster-graphql/dagster_graphql/schema/schedules/schedules.py

```diff
@@ -41,6 +41,12 @@ class GrapheneSchedule(graphene.ObjectType):
     futureTick = graphene.NonNull(
         GrapheneDryRunInstigationTick, tick_timestamp=graphene.NonNull(graphene.Int)
     )
+    ticksFromTimestamp = graphene.NonNull(
+        graphene.List(graphene.Float),
+        start_timestamp=graphene.Float(),
+        upper_limit=graphene.Int(),
+        lower_limit=graphene.Int(),
+    )
```

Comment:
what other names did you consider besides `ticksFromTimestamp` ?

Reply:
another is potentialTickTimestamps, timestampsInRange

Reply:
I think i like `potentialTickTimestamps` most